### PR TITLE
Add Norwegian Bokmål localizations to Sileo

### PR DESCRIPTION
**Pull request by worldlydev**
Not sure if I did this correctly but all my translations are included in this PR.

Suggestion:

- Perhaps use Crowdin to handle localizations now that Sileo is open-source? Makes it super easy to update strings for non-GitHub oriented people like me ;) I could open a GitHub issue for it if the team is at all open to that.

Question:

- If I were to translate to Norwegian Nynorsk as well would I name the lproj: nn.lproj or nn-NO.lproj?

### CODE REVIEW

**Comment by worldlydev** [comment]
The Localizable.stringsdict is the part I'm not sure about btw but I may have messed up something else as well. Feel free to @ me.

File: Sileo/Sileo/nb.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  Created by AppleBetas on 2018-12-27.
+  Translated to Norwegian Bokmål by : WorldlyDev (2020-04-21).
+  Copyright © 2018 CoolStar. All rights reserved.
+ 
+ NORWEGIAN BOKMÅL LOCALISATIONS
+*/
+
+// Credit Information (optional, uncomment to specify)
+//"__LocalizationCredit.Title" = "Sileo oversatt til Norsk Bokmål av WorldlyDev";
+//"__LocalizationCredit.URL" = "https://twitter.com/worldlydev"; // (optional)
```
Comment (kirb) [inline Sileo/Sileo/nb.lproj/Localizable.strings:14]: Hi, thanks for submitting translations! Have you tested these with the current (1.7.4 I think) release of Sileo?

Comment (worldlydev) [reply Sileo/Sileo/nb.lproj/Localizable.strings]: Just fixed this.

I haven't been able to test this as I'm currently on Unc0ver (11 pro, 13.3) and don't have Sileo as my default. If you give me a couple of minutes I will be able to see if it all looks good.

Was the Localizable.stringsdict done correctly? Not really sure what it is to be honest. This kind of coding goes way above my level of understanting :)

**Comment by worldlydev** [comment]
Should be ready to merge now :)

**Comment by MajiOSx** [comment]
Sileo

**Pull request closed without merging**